### DISCUSSION TITLE
Make DNS request before check in

### DISCUSSION THREAD

**Discussion by Bubba8291** [jdholtz/auto-southwest-check-in#263] [Ideas]
I have an idea for speeding up the check in. Would if the script makes the DNS request to mobile.southwest.com like a minute before the check in and stores it. I am not sure if it will make a difference, but every millisecond counts.

**Comment by babehboi** [comment]
But I believe today it starts 5s before the start time? What would making the DNS request do? Just wondering?

**Reply by jdholtz** [reply]
In addition to this, because the script checks the Southwest website for various things (flight prices, new reservations, ensuring scheduled check-ins are still valid, etc.) it is very possible that at the time of check-in your computer would just use the cached DNS record rather than even re-querying it.
